cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -116,6 +118,61 @@ protected void configure() {
     };
   }
 
+  public Module getPreviewModules(final DatasetFramework remoteDatasetFramework) {
```

Comment:
I think it's better to have preview related stuff under preview packages/classes instead of sitting inside normal classes.